src/main/webapp/web.xml

```diff
@@ -105,4 +105,40 @@
         <servlet-name>deleteManufacturer</servlet-name>
         <url-pattern>/manufacturers/delete</url-pattern>
     </servlet-mapping>
+
+    <servlet>
+        <servlet-name>loginDriver</servlet-name>
+        <servlet-class>mate.controller.driver.LoginController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>loginDriver</servlet-name>
+        <url-pattern>/login</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>logoutDriver</servlet-name>
+        <servlet-class>mate.controller.driver.LogoutController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>logoutDriver</servlet-name>
+        <url-pattern>/logout</url-pattern>
+    </servlet-mapping>
+
+    <servlet>
+        <servlet-name>getCarsByDriver</servlet-name>
+        <servlet-class>mate.controller.driver.GetMyCurrentCarsController</servlet-class>
+    </servlet>
+    <servlet-mapping>
+        <servlet-name>getCarsByDriver</servlet-name>
+        <url-pattern>/cars/all</url-pattern>
+    </servlet-mapping>
```

Comment:
```suggestion
    <servlet>
        <servlet-name>getMyCars</servlet-name>
        <servlet-class>mate.controller.driver.GetMyCurrentCarsController</servlet-class>
    </servlet>
    <servlet-mapping>
        <servlet-name>getMyCars</servlet-name>
        <url-pattern>/cars/current</url-pattern>
    </servlet-mapping>
```

Reply:
/cars/all looks like you want to get all cars

Reply:
++